noir_stdlib/src/option.nr

```diff
@@ -112,19 +116,31 @@ impl<T> Option<T> {
 
     /// If self is Some, return self. Otherwise, return `other`.
     pub fn or(self, other: Self) -> Self {
-        if self._is_some { self } else { other }
+        if self._is_some {
+            self
+        } else {
+            other
+        }
     }
 
     /// If self is Some, return self. Otherwise, return `default()`.
     pub fn or_else<Env>(self, default: fn[Env]() -> Self) -> Self {
-        if self._is_some { self } else { default() }
+        if self._is_some {
+            self
+        } else {
+            default()
+        }
```

Comment:
This feels like we're setting `single_line_if_else_max_width` too low by default. I'm imagining that we're measuring the width differently than with the previous formatter.

Reply:
Oh, that's intended. `if` as a statement is always formatted in multiple lines, while `if` as an expression can be inlined. It's the same in Rust if you try it.

I _think_ the reason is that if you have code like this:

```rust
if 1 {
    2 
} else {
    3
}
(4, 5)
```

then the formatter would change it to this:

```rust
if 1 { 2 } else { 3 } (4, 5)
```

and it looks like a method call on that if. I think we also had a similar bug where the formatter would do that, and it was because `if` was always parsed as an expression, never as a statement. Once we changed that it got fixed.

Reply:
Ok, sounds good.